src/http/article.controller.js

```diff
@@ -44,6 +44,35 @@ class ArticleController extends BaseController {
         res.status(200)
         return res.render('article/index', { title: article.title, body: article.body, tags: article.tags });
     }
+
+    async update(req, res) {
+        const data = req.body
+
+        // Temporary validation, you're really taking your time with it huh
+        if (
+            typeof req.params.id !== "string" || !req.params.id || !data || typeof data.title !== "string" || data.title.length < 3 || data.title.length > 250 ||
```

Comment:
``req.params.id`` should be validated separately since it is part of the route and if it doesn't exist in our app it should be considered "Not found".